integration-cli/docker_api_stats.go

```diff
@@ -0,0 +1,53 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/docker/docker/api/types"
+	"github.com/go-check/check"
+	"strings"
+	"time"
+)
+
+func (s *DockerSuite) TestCliStatsNoStreamGetCpu(c *check.C) {
+	dockerFile := fmt.Sprintf("FROM busybox\nRUN echo \"#/bin/sh\" > /bin/test.sh && echo \"while true;do\" >> /bin/test.sh && echo \"echo \"Hello world\"\" >> /bin/test.sh && echo \"done\" >> /bin/test.sh\nRUN chmod +x /bin/test.sh\nCMD /bin/test.sh")
```

Comment:
Thanks for the test, but some notes here:
I would prefer to not build an image since it is unnecessary to perform the test, and adds a significant amount of time to the actual test.

```
out, _ := dockerCmd(c, "run", "-d", "busybox", "/bin/sh", "-c", "while true; do echo 'hello'; sleep 0.01; done")
```
That ought to be sufficient to create the required test and also not completely saturate the cpu.